calculus.tex

```diff
@@ -4,21 +4,25 @@
 
 Derivatives can be computed with the \verb|diff| function.
 \begin{verbatim}
+>>> x = symbols("x")
```

Comment:
The text at the top of the paper says that all examples will assume `from sympy import *` and `x, y, z = symbols('x y z')`.